scripts/windows/setup/IotEdgeSecurityDaemon.ps1

```diff
@@ -309,46 +309,52 @@ function Initialize-IoTEdge {
     Set-StrictMode -Version 5
 
     if (-not (Test-EdgeAlreadyInstalled)) {
+        $error = ('IoT Edge is not yet installed. ' + $InstallMessage)
         Write-HostRed
-        Write-HostRed ('IoT Edge is not yet installed. ' + $InstallMessage)
-        throw
+        Write-HostRed $error
+        throw $error
```

Comment:
but why do we need to throw again with same error message? error message should already print out by Write-HostRed.

Reply:
*Write-Host* and *Write-HostRed* do not print anything in the DevOps logs, not sure why. That's the reason to have the same error message thrown as an exception (that makes it into the DevOps logs).

Reply:
is it because iotedgequickstart swallow all logs from IotEdgeSecurityDaemon.ps1; we should try to fix it if that's the case.

Reply:
Good point.